src/events/event.cpp

```diff
@@ -725,6 +728,10 @@ EventCameraPathFinished::EventCameraPathFinished(const SceneGraphNode* origin_,
     , destination(temporaryString(destination_->identifier()))
 {}
 
+EventCameraMovedPosition::EventCameraMovedPosition()
+    : Event(Type)
+{}
+
```

Comment:
A few things missing in this file (see comment in the top of the `Event` struct
`void log(int i, const EventCameraMovedPosition& e)`  needs to be defined

The `switch/case` statement in the `std::string_view toString(Event::Type type)` function needs to be updated to check for the new type.